br/pkg/restore/split_client.go

```diff
@@ -168,10 +168,6 @@ func (c *pdClient) SplitRegion(ctx context.Context, regionInfo *RegionInfo, key
 		return nil, errors.Trace(err)
 	}
 	if resp.RegionError != nil {
-		log.Error("fail to split region",
-			logutil.Region(regionInfo.Region),
-			logutil.Key("key", key),
-			zap.Stringer("regionErr", resp.RegionError))
 		return nil, errors.Annotatef(berrors.ErrRestoreSplitFailed, "err=%v", resp.RegionError)
```

Comment:
If possible, I think we'd better also add the region info and the key into the error for debugging.